libs/isograph-react/src/core/read.ts

```diff
@@ -797,6 +798,104 @@ export function readLinkedFieldData(
     };
   }
   const targetId = link;
+  if (typeof field.refetchQuery === 'number') {
+    root = link;
```

Comment:
nit: this is a bit confusing. Maybe it will be a bit clearer to define a new variable, `clientPointerTargetId` or somthing?

Reply:
well, we have that with `targetId` already!